HarpGuru/components/index.tsx

```diff
@@ -44,8 +25,14 @@ enum MenuStates {
   NoMenu,
 }
 
+// TODO: this is messy. I have exported the setGlobalReactNState
+// string as a convenient way to load the file which creates the
+// global state and simultaneously report on what the state was
+// set as. At the moment I don't actually have anywhere I want to
+// report it so this is a loose thread.
+export const initialReactNState = setGlobalReactNState
```

Comment:
What am I going to do about this. It's real hacky, but I haven't turned up anything helpful on a very quick web search, so perhaps I just need to make sure this is also identified for attention in an existing or new issue.

Reply:
I'm tempted to raise a dedicated ticket for this, but I'm going to place it under https://github.com/js-jslog/harpnative/issues/45 along with everything else.

Reply:
In the end this was addressed in https://github.com/js-jslog/harpnative/pull/48